src/finn/custom_op/fpgadataflow/matrixvectoractivation.py

```diff
@@ -385,6 +455,25 @@ def dsp_estimation(self):
         else:
             mult_dsp = 0
         return int(mult_dsp)
```

Comment:
If the resource estimation depend on the implementation style (HLS or RTL), please move the estimation functions in the respective custom ops.